docs/examples/tar_and_transfer.rst

```diff
@@ -1,6 +1,50 @@
 Tar and Transfer
 ----------------
 
+This example highlights the usage of the Tar and Transfer Gladier tools in a single flow.
+Start the workflow by overriding the Gladier Client with a list of the
+tools you want to use. In this case, we want to use the Tar, and Transfer tools.
 
+.. code-block:: python
 
-Tar and Tranfer files
\ No newline at end of file
+   @generate_flow_definition
+   class TarAndTransfer(GladierBaseClient):
+       gladier_tools = [
+       'gladier_tools.posix.Tar',
+       'gladier_tools.globus.Transfer',
+        ]
+    
+The ``@generate_flow_definition`` decorator takes the flow definitions of the individual tools and constructs a flow definition for the new flow, so we do not have 
+to define a custom flow definition.
```

Comment:
Nice description here. It might also be good to link to the flow generation doc. Additionally, you could mention the user can see the entire flow by using ` pprint(tat.flow_definition)`